server/src/controllers/Events/resolver.ts

```diff
@@ -740,21 +785,12 @@ ${unsubscribeOptions}`,
 
     updateReminders(event, update.start_at);
 
-    if (hasVenueChanged(data, event)) {
-      const { body, subject } = await buildEmailForUpdatedEvent(data, event);
-      batchSender(function* () {
-        for (const { user } of event.event_users) {
-          const email = user.email;
-          const unsubscribeOptions = getEventUnsubscribeOptions({
-            chapterId: event.chapter_id,
-            eventId: event.id,
-            userId: user.id,
-          });
-          const text = `${body}<br>${unsubscribeOptions}`;
-          yield { email, subject, text };
-        }
-      });
-    }
+    hasVenueChanged(data, event) ||
+      (hasDateChanged(data, event) &&
+        createEmailForSubscribers(
+          buildEmailForUpdatedEvent(data, event),
+          event,
+        ));
```

Comment:
This is quite a weird pattern. Normally if you want to do something conditionally

```
if(cond) {
  // do stuff
}
// or
if (cond) // od stuff
```
is typical, but

```
cond && // do stuff
```

is not.